napari/layers/points/points.py

```diff
@@ -1799,3 +1794,19 @@ def _copy_data(self):
 
         else:
             self._clipboard = {}
+
+
+def _fix_data_points(
```

Comment:
Can you add a doc string here? Can you also add the method to `_points_utils.py` and import it, instead of putting it at the bottom of the `points.py` file.